src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -180,6 +180,20 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
                     logInfo(WM_VULNSCAN_LOGTAG, "Processing message");
                     TMessageProcessor::processMessage(message, orchestrationLambda);
                     logInfo(WM_VULNSCAN_LOGTAG, "Message processed");
+
+                    // Compact the database
+                    logInfo(WM_VULNSCAN_LOGTAG, "Compacting CVE5 database");
+                    m_cvesDatabase->compactDatabase();
```

Comment:
Please, use the method with bzip2 compression

Reply:
Yes sorry! I forgot it!